### PR TITLE
Selinux policy and docs

### PR DESCRIPTION
**Pull request by croepha**
Please consider adding these changes

### CODE REVIEW

File: docs/selinux.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+```
+checkmodule -M -m -o docker-openvpn.mod docker-openvpn.te
+semodule_package -o docker-openvpn.pp -m docker-openvpn.mod
+sudo semodule -i docker-openvpn.pp
+```
+
+# Still having issues?
+
+In January 2016, Fedora bases systems got an update that fixed an issue for labeling namespaced net objects under /proc
```
Comment (ypid) [inline docs/selinux.md]: "Fedora bases" → "Fedora based"

**Comment by croepha** [comment]
Fixed, thanks @ypid

**Comment by croepha** [comment]
Please let me know if further explanation is required, or if there is anything else I can do to help get this merged

**Comment by kylemanna** [comment]
Awesome, thanks!

**Pull request merged**